Sources/ApolloCodegenLib/ApolloCodegen.swift

```diff
@@ -124,6 +124,37 @@ public class ApolloCodegen {
       )
     }
   }
+  
+  public static func generateOperationManifest(
+    with configuration: ApolloCodegenConfiguration,
+    withRootURL rootURL: URL? = nil,
+    fileManager: ApolloFileManager = .default
+  ) throws {
+    let configContext = ConfigurationContext(
+      config: configuration,
+      rootURL: rootURL
+    )
+    
+    try validate(configContext)
+    
+    let compilationResult = try compileGraphQLResult(
+      configContext,
+      experimentalFeatures: configuration.experimentalFeatures
+    )
+    
+    try validate(configContext, with: compilationResult)
+    
+    let ir = IR(compilationResult: compilationResult)
+    
+    var operationIDsFileGenerator = OperationManifestFileGenerator(config: configContext)
```

Comment:
Could this be a `let` constant too?

Reply:
So `collectOperationIdentifier` is a mutating function so this does still need to be a var.